spec/spec_helper.rb

```diff
@@ -8,7 +8,7 @@
 Dir["#{File.dirname(__FILE__)}/support/**/*.rb"].each {|f| require f}
 
 RSpec.configure do |config|
-  
+  config.expect_with :rspec
```

Comment:
The test suite is passing, but I needed to add `config.expect_with :rspec` to spec/spec_helper.rb. I shouldn't need to do this.
If you remove the `expect_with :rspec`, you get this error while running the tests: 
`NoMethodError: undefined method 'expect' for #<RSpec::ExampleGroups::AppConfig:0x000000013263f898>`
@jalada Any idea what's going on here?